lib/kube/proxy/resource_deletecollection.go

```diff
@@ -174,304 +177,304 @@ func (f *Forwarder) handleDeleteCollectionReq(req *http.Request, sess *clusterSe
 		items, err := deleteResources(
 			params,
 			types.KindKubePod,
-			arrayToPointerArray(o.Items),
+			slices.SliceToPointerSlice(o.Items),
```

Comment:
Since the package is called `slices`, this stutters a bit.

What do you think about the following names instead?

- `slices.ToPointers`: convert `[]T` to `[]*T`
- `slices.FromPointers`: convert `[]*T` to `[]T`

Reply:
Let's call this next commit a Diet Slice.